data/json/items/gunmod/mount.json

```diff
@@ -3,7 +3,7 @@
     "id": "grip_mount",
     "type": "GUNMOD",
     "name": { "str": "replaceable furniture kit" },
-    "description": "This is a kit consisting of various steel and plastic parts; when installed, it would permanently modify and partially replace some of the weapon's furniture so that it could be easily changed if needed.  This allows installing any kind of more advanced grips or other furniture.",
+    "description": "A kit consisting of various steel and plastic parts.  When installed, it will permanently modify and partially replace some of the weapon's furniture so that it can be easily changed if needed.  This allows installing any kind of more advanced grips or other furniture.",
```

Comment:
```suggestion
    "description": "A kit consisting of various steel and plastic parts.  When installed, it will permanently modify and partially replace some of the firearm's furniture so that it can be easily changed if needed.  This allows installing any kind of more advanced grips or other furniture.",
```